cmd/branchProtection.go

```diff
@@ -80,9 +80,15 @@ func (verifier BranchProtectionVerifier) getRepo() *github.Repository {
 }
 
 func (verifier BranchProtectionVerifier) addExistingChecksToRequest(existingProtection *github.Protection, protectionRequest github.ProtectionRequest) {
-	if existingProtection == nil || existingProtection.RequiredStatusChecks == nil || existingProtection.RequiredStatusChecks.Contexts == nil {
+	if existingProtection == nil || existingProtection.RequiredStatusChecks == nil || existingProtection.RequiredStatusChecks.Contexts == nil || len(existingProtection.RequiredStatusChecks.Contexts) == 0 {
 		return
 	}
+	if protectionRequest.RequiredStatusChecks == nil {
+		protectionRequest.RequiredStatusChecks = &github.RequiredStatusChecks{
+			Strict:   true,
+			Contexts: []string{},
+		}
```

Comment:
I'd expect that that's already the default. Maybe you could just wrap the for loop with:
```
if protectionRequest.RequiredStatusChecks != nil {
```

Reply:
Sorry, that's not possible with the GH API